yt/visualization/volume_rendering/scene.py

```diff
@@ -369,6 +369,8 @@ def save_annotated(
         sigma_clip: Optional[float] = None,
         render: bool = True,
         tf_rect: Optional[List[float]] = None,
+        *,
+        label_fontsize: int = 10,
```

Comment:
given that it ends up being passed to `matplotlib.axes.Axes.tick_params`, the proper type is actually `Union[float, str]` (`'large'` also works) (see https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.tick_params.html)

Reply:
updated